package.json

```diff
@@ -20,7 +20,7 @@
     "view-cover": "opn coverage/index.html"
   },
   "dependencies": {
-    "body": "^4.6.0",
+    "body": "Raynos/body#5308bb4",
```

Comment:
we cannot ship github links into prod. We'll need to wait for a new version of body.